bin/audit-svg-colors.js

```diff
@@ -0,0 +1,282 @@
+#!/usr/bin/env node
+
+/**
+ * The script looks for all colors used in the SVG images found in the repository
+ * and suggests Color Studio replacements for all non-standard values it finds.
+ *
+ * Please note that the suggestions are automattic and therefore not perfect
```

Comment:
Gets me every time 😂 

```suggestion
 * Please note that the suggestions are automatic and therefore not perfect
```

Reply:
Yeah…